docs/src/main/tut/docs.md

```diff
@@ -3,38 +3,56 @@ layout: docs
 title: Getting Started
 ---
 
-# Get started
+# Getting started
 
 WIP: Import
 
 ```tut:silent
 import github4s.Github
 ```
 
+In order for github4s to work in both JVM and scala-js environments, you'll need to place different implicits in your scope:
+
+```tut:silent
+object JVMProgram extends github4s.ImplicitsJVM {
```

Comment:
we should provide objects for implicits imports such as `github4s.jvm.implicits._` beside traits for aggregation. In most use cases folks are going to want an import not aggregating a trait.